src/adapter/src/coord/sequencer.rs

```diff
@@ -3978,6 +3984,31 @@ impl Coordinator {
         }
         Ok(ops)
     }
+
+    /// Installs the in-memory state for the cluster linked to the specified
+    /// object after a create operation, if such a linked cluster exists.
```

Comment:
I think these will not only install the in-memory state, but may also call the controller functions to create the physical resource. Should we include that in the comment.

Reply:
Yeah, good call. It's historically been "on-disk" and "in-memory", but you're totally right that what I think of as "in-memory" involves literally turning on computers ... so yeah, confusing. Updated!